cypress/e2e/tests/administration/jira-connection/interlinked.test.ts

```diff
@@ -0,0 +1,152 @@
+import { JiraCredentials } from "../../../models/administration/credentials/JiraCredentials";
+import { MigrationWave } from "../../../models/migration/migration-waves/migration-wave";
+import * as data from "../../../../utils/data_utils";
+import { Assessment } from "../../../models/migration/applicationinventory/assessment";
+import { jiraAlert, jiraTable } from "../../../views/jira.view";
+
+describe(["@tier2"], "Jira connection negative tests", () => {
+    const expectedToFail = true;
+    const useTestingAccount = true;
+    const isSecure = false;
+    let jiraBasicCredential: JiraCredentials;
+    let jiraBasicCredentialInvalid: JiraCredentials;
+    let jiraCloudConnectionData: JiraConnectionData;
+    let jiraCloudConnectionDataIncorrect: JiraConnectionData;
+    let jiraCloudConnection: Jira;
+    let jiraCloudConnectionIncorrect: Jira;
+    let applicationList: Array<Assessment> = [];
+    const now = new Date();
+    now.setDate(now.getDate() + 1);
+    const end = new Date(now.getTime());
+    end.setFullYear(end.getFullYear() + 1);
+    let migrationWave: MigrationWave;
+
+    before("Login and create required credentials", function () {
+        login();
+        jiraBasicCredential = new JiraCredentials(
+            getJiraCredentialData(CredentialType.jiraBasic, useTestingAccount)
+        );
+
+        jiraBasicCredential.create();
+
+        // Defining Jira connection data with correct credentials
+        jiraCloudConnectionData = getJiraConnectionData(
+            jiraBasicCredential,
+            isSecure,
+            useTestingAccount
+        );
+
+        jiraCloudConnection = new Jira(jiraCloudConnectionData);
+
+        // Defining and creating dummy credentials to be used further in tests
+        jiraBasicCredentialInvalid = new JiraCredentials(
+            getJiraCredentialData(CredentialType.jiraBasic, !useTestingAccount)
+        );
+
+        jiraBasicCredentialInvalid.create();
+
+        // Defining Jira connection data with incorrect credentials
+        jiraCloudConnectionDataIncorrect = getJiraConnectionData(
+            jiraBasicCredentialInvalid,
+            isSecure,
+            useTestingAccount
+        );
+
+        jiraCloudConnectionIncorrect = new Jira(jiraCloudConnectionDataIncorrect);
+
+        applicationList = createMultipleApplications(2);
+    });
+
+    beforeEach("Interceptors", function () {
+        cy.intercept("DELETE", "/hub/migrationwaves*/*").as("deleteWave");
+    });
+
+    it("Creating Jira connection with incorrect credentials", () => {
+        jiraCloudConnectionIncorrect.create();
+        jiraCloudConnectionIncorrect.validateState(expectedToFail);
+    });
+
+    it("Bug MTA-1014 | Trying to remove Jira connection used by wave", () => {
+        const issueType = JiraIssueTypes.task;
+        let projectName = "";
+        jiraCloudConnection.create();
+        jiraCloudConnection.validateState(!expectedToFail);
```

Comment:
This validation is not required here as it is already done in CRUD .
and again jiraconnection.create() should not send any argument if it is expected to pass .

Reply:
This is needed here cause connection becomes green not immediately and I need to wait for that, so I thought that is a good way to achieve this

Reply:
Ok I thought we are checking alert message here . Sounds good :+1: